src/agent/communicator/include/ihttp_socket.hpp

```diff
@@ -8,6 +8,9 @@
 
 namespace http_client
 {
+    /// @brief The socket timeout in seconds
+    constexpr int SOCKET_TIMEOUT_SECS = 2;
```

Comment:
```suggestion
    constexpr int SOCKET_TIMEOUT_SECS = 10;
```
I think 2 seconds is a small number, what do you think?